tools/provision.py

```diff
@@ -207,6 +207,7 @@ def main(options):
     run(["sudo", "chown", "%s:%s" % (user_id, user_id), EMOJI_CACHE_PATH])
     run(["tools/setup/emoji/build_emoji"])
     run(["scripts/setup/generate_secrets.py", "--development"])
+    run(["tools/update-authors-json", "--use-fixture"])
```

Comment:
(Otherwise, the tests will fail for anyone following our manual setup instructions.  I think it's possible that moving this to a `setUp` method on the test class might actually be the cleanest approach from an overall complexity perspective.)